Watchman.Discord/Areas/Initialization/Controllers/InitializationController.cs

```diff
@@ -103,5 +112,32 @@ private async Task ReadServerMessagesHistory(Contexts contexts)
             var messagesService = _messagesServiceFactory.Create(contexts);
             await messagesService.SendResponse(x => x.ReadingHistoryDone(), contexts);
         }
+
+        private async Task SafeRolesInit(DiscordServerContext server)
+        {
+            var defaultSafeRoles = new List<Role>
+            {
```

Comment:
hmm myśle czy chcemy to dawać do init 
bo nie chcemy obsługiwać serwerów tylko programistycznych

Reply:
Pomyślałem, żeby robić automatycznie taką postawę ról, no ale można zrobić to np. pod osobną komendą, aby nie trzeba ręcznie dodawać kilku/kilkunastu roli.